packages/irt/package.py

```diff
@@ -27,3 +28,12 @@ class Irt(CMakePackage):
     version("1.0.0", sha256="55746700a477ed4decbdadbc008b43f370071cdd699452b96d7daa1dbc4ee28d")
 
     depends_on("root@6: +root7")
+
+    def cmake_args(self):
+        args = [
+            "-DEVALUATION=OFF",
+            "-DDELPHES=OFF",
+        ]
+        if self.spec.satisfies('@1.0.6:'):
```

Comment:
```suggestion
        if self.spec.satisfies('+root_io'):
```

Reply:
Or even:
```python
args.enable_variant("IRT_ROOT_IO", "root_io")
```

Reply:
But I am on my phone, unable to look up the exact syntax on that last one .

Reply:
https://spack.readthedocs.io/en/latest/build_systems/cmakepackage.html#adding-flags-to-cmake

It's `define_from_variant`.